charts/jenkins/Chart.yaml

```diff
@@ -40,7 +40,7 @@ annotations:
     - name: jenkins
       image: docker.io/jenkins/jenkins:2.504.3-jdk21
     - name: k8s-sidecar
-      image: docker.io/kiwigrid/k8s-sidecar:1.30.5
+      image: docker.io/kiwigrid/k8s-sidecar:1.30.3
```

Comment:
you also need to update https://github.com/jenkinsci/helm-charts/blob/ce3b1c74ed81b03b51baf697ef4b142a806eb654/charts/jenkins/values.yaml#L573 and run `.github/helm-docs.sh`

Reply:
Updated